### DISCUSSION TITLE
There is any tutorial of how to integrate a live camera view ?

### DISCUSSION THREAD

**Discussion by BrunoDC-dev** [nasa/openmct#7336] [Community Q&A] [answered]
i am trying to integrate the cameras of my dron in live time

**Comment by akhenry** [comment]
@davetsay

**Reply by davetsay** [reply, marked as answer]
@BrunoDC-dev , at the moment, there are two ways to achieve streaming a video.

The `Web Page` object is available as a creatable object, and allows embedding the contents of a url in an iframe. 

Open MCT also has an Imagery plugin. In realtime mode, the imagery view will update to show the most recent image, or frame in your case, and also have a thumbnail strip to access previous images. We have seen performant implementations of this with frequency of multiple images per second.

If neither of those work for your use case, you would have to implement a custom plugin.
